examples/02-plot/lookup-table.py

```diff
@@ -0,0 +1,236 @@
+"""
+.. _lookup_table_example:
+
+Anti-Aliasing
+~~~~~~~~~~~~~
+Demonstrate the usage of a lookup table within PyVista
+
+The :class:`pyvista.LookupTable` can be used to have fine-tuned control over
+the mapping between a :class:`pyvista.DataSet`'s scalars and RGBA colors.
+
+"""
+
+import pyvista as pv
+from pyvista import examples
+
+pv.set_plot_theme('document')
+
+# download an example dataset
+bracket = examples.download_fea_bracket().cell_data_to_point_data()
+bracket
+
+
+###############################################################################
+# Default Color Map - Lookup Table
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# First, let's plot using the default color map, "viridis". Internally, PyVista
+# will automatically create a lookup table to map the scalars (stored here
+# within `point_data`, to RGBA colors. This is shown here as a nested attribute
+# to the :class:`pyvista.DataSetMapper` and it has a helpful ``repr`` method:
+
+pl = pv.Plotter()
+actor = pl.add_mesh(bracket)
+actor.mapper.lookup_table
+
+
+###############################################################################
+# Plot the Lookup Table
+# ~~~~~~~~~~~~~~~~~~~~~
+# You can also plot lookup table to see the mapping between the scalar values
+# (here, between 0 and 23.3) and RGBA colors.
+
+pl = pv.Plotter()
+actor = pl.add_mesh(bracket)
+actor.mapper.lookup_table.plot()
+
+
+###############################################################################
+# Plot the DataSet
+# ~~~~~~~~~~~~~~~~
+# Lets plot the dataset using the automatically generated lookup table.
+
+pl = pv.Plotter()
+pl.add_mesh(bracket)
+pl.show()
+
+
+###############################################################################
+# Create a Custom Lookup Table using a matplotlib Color Map
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Here we create a lookup table with a narrow table range (same as ``clim``)
+# and color values above and below the range.
+
+lut = pv.LookupTable(cmap='magma')
+lut.table_range = (5, 15)
+lut.below_range_color = pv.Color('grey', opacity=0.5)
+lut.above_range_color = 'r'
+lut.plot()
+
+
+###############################################################################
+# Plot the bracket with the custom colormap
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# You can set assign the lookup table when using ``add_mesh``.
+
+
+pl = pv.Plotter()
+actor = pl.add_mesh(bracket, lookup_table=lut, lighting=False)
+pl.show()
```

Comment:
Is this supposed to look like this? Feels off to me...

<img width="624" alt="Screen Shot 2022-09-21 at 9 22 33 AM" src="https://user-images.githubusercontent.com/22067021/191544975-6756bc2c-40d3-4c91-a596-e2ff3c62a3a0.png">

Reply:
Yep, anything below a certain stress is opaque. It's not perfect and it's a weird cmap, but it's just a demo.